templates/assets/2018-12-26-kerstkampen

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
---
layout: post
title:  "Basketbal voor de jeugd tijdens de kerstperiode"
cover: /BasketLummen/Website/_content/albums/2018-10-13-Basket-Lummen-DSEA-Sint-Katelijne-Waver-DSEA/_thumbnails/IMG_6412.jpg
date:   2018-10-25 20:00:00
description: Basketbal voor de jeugd, ook tijdens de kerstperiode!.
permalink: /news/vervang-dit-door-de-bestandsnaam-zonder-extensie/
---

Ook tijdens de kerstperiode zullen de jeugdspelers van Basket Lummen leerrijke ervaringen kunnen opdoen. Er is niet alleen het FROS-kamp, met coaches van Basket Lummen, maar ook het Bounce Camp Hans Vanwijn, een voormalig speler van Basket Lummen. Verder volgt er nog een evenement (Brecht)..... details...?

## FROS-kamp

Locale coaches, zoals Jolien Wouters, Janne Wouters en Caro Swolfs, hebben een ijzersterk concept voor de beginnende en gevorderde basketter. Het kamp is al jarenlang een vaste waarde in Limburg en bundelt heel wat ervaring. Het wordt georganiseerd door Bart Wouters, die onze DSE-A-ploeg ondersteunt.

#### Voor wie?
Voor jongens en meisjes vanaf 7 jaar, met minimaal 1 jaar basketervaring.

#### Praktisch
Op 26, 27, en 28 december in de sporthal van Lummen, van 09h00 tot 16h00.

#### Inschrijven
In de bijgevoegde folder vind je alle informatie om in te schrijven. |link naar .pdf-bestand toevoegen|

## Bounce Camp Hans Vanwijn

Hans Vanwijn, een voormalig speler van Basket Lummen, leidt een aantal coaches, die hun ervaring bij Limburg United zullen gebruiken voor onze jeugdspelers. 

#### Voor wie?
Voor jongens en meisjes tussen 10 en 16 jaar, met minimaal 1 jaar basketervaring.

#### Praktisch
Op 2, 3 en 4 januari, Bogaarsveldstraat in Beringen.

#### Inschrijven
Via deze link kan je de inschrijving regelen: [www.hansvanwijn.be](https://www.hansvanwijn.be/kampen/)

## evenement Brecht 

details ontbreken nog

#### Voor wie?
Voor 

#### Praktisch
Op 

#### Inschrijven
Via

Reply:
Dit is het voorstel voor de tekst (maar deze is natuurlijk nog niet volledig).

Reply:
Je hebt de tekst nogmaals als commentaar op de eerste regel van de tekst zelf gezet, was dat de bedoeling, of had je de bedoeling wijzigingen door te voeren aan je tekst?

Voor wijzigingen is hier de handleiding: https://github.com/BasketLummen/Communicatie/blob/master/processes/voorstellen-wijzigen.md

Reply:
Ik vind alleen een fork op BasketLummen/Website uit mei 2018, maar niet BasketLummen/Communicatie.  Deze zal ik dan eerst moeten aanmaken, neem ik aan. Zal dit morgen proberen - hopelijk hebben we dan (via Slack) wat basisinformatie over Brechts evenement en kunnen we de tekst vervolledigen, met een foto erbij.

Reply:
Dit zie ik wel:
![image](https://user-images.githubusercontent.com/39502859/47187407-41c52780-d334-11e8-9190-e1cbd286752c.png)

Reply:
Ah, nu snap ik het probleem... de PR moet je maken op de repository van de website, niet hier ;)

Ik zal het morgen vroeg overzetten naar de andere repository

Reply:
We gaan geen info krijgen over dat 3e kamp hoor, want het is nog niet in detail beslist.